includes/html/api_functions.inc.php

```diff
@@ -513,27 +515,27 @@ function maintenance_device(Illuminate\Http\Request $request)
         return api_error(404, "Device $hostname not found");
     }
 
-    if (! $request->json('duration')) {
+    if (! $data['duration']) {
```

Comment:
this cold fail if the key is missing needs to use one of: isset, empty, array_key_exists, ?? etc.

Reply:
There are a few spots below also.  One advantage of the ->json calls was we did not need to check isset.

Reply:
yeah, I moved over to the $data structure like all the rest as these two functions were literally the only two using ->json calls like that in checks from 4+ years ago, assumed it was not the preferred way.